cmd/divide.go

```diff
@@ -58,7 +57,7 @@ var (
 )
 
 func init() {
-	divideCmd.Flags().Int64SliceP("users", "u", []int64{}, "Divides by desired users instead of equal divisions. Say you want your network divided into 32 users, 10 users, and 17 users: 'cidr divide <network> 32,10,17' ")
+	divideCmd.Flags().Int64SliceP("hosts", "H", []int64{}, "Divides by desired hosts instead of equal divisions. Say you want your network divided into 32 hosts, 10 hosts, and 17 hosts: 'cidr divide <network> 32,10,17' ")
```

Comment:
Perhaps we can move to Int64Slice, instead of Int64SliceP to force a user to use `--hosts` instead of `-H` which might confuse them as `-h` is for help

Reply:
I'll let you have that call, as I am a big fan of shorthands for CLI tools. When they're thrown in scripts, that's when I prefer the longer syntax.